friskby/GW/air_in_bergen.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+
+import requests
+import timestring
+#  import re
+#  import time
+from datetime import date
+import psycopg2
+from friskby.GW.friskby_gw import FriskByGW
+
+sensor_types = ['_PM10', '_PM25']
+
+GW = FriskByGW()
+
+luft_map = {"BG_1": "http://www.luftkvalitet.info/home/"
+                          "overview.aspx?type=2&topic=1&id=%7b751808f5-d561-4737-9185-4ecc0e834975%7d", # LODDEFJORD
+            "BG_2": "http://luftkvalitet.info/"
+                             "home/overview.aspx?type=2&topic=1&id=%7b4ff685c1-ad51-4468-b2fc-08345d11f447%7d", # DANMARKSPLASS
+            "BG_3": "http://www.luftkvalitet.info/home/"
+                         "overview.aspx?type=2&topic=1&id=%7b5b0ff070-e6e6-4f60-88a3-bd923ac3a7e6%7d", # RAADHUSET
+            "BG_4": "http://www.luftkvalitet.info/home/"
+                      "overview.aspx?type=2&topic=1&id=%7bceade2ac-e62f-4e50-af7c-347e402fff27%7d" # AASANE
+
+            }
+# keys for parsing data
+komponent_id = ["PM10", "PM2.5"] #, "NO2", "O3"]
+
+key = "00001111-2222-3333-4444-555566667777"
```

Comment:
If were the least bit worried of security the key should not be embedded in the source code, but rather come in as environment variable - ignore that for now.